example/test.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -e
+
+# Set a trap on errors to make it clear when tests have failed
+trap '{ set +x; echo; echo FAILED; echo; } >&2' ERR
+
+set -x
+
+# Bring up the DB and AMQP first
+docker-compose up -d db amqp && sleep 5
+
+# Bring everything else up
+docker-compose up -d && sleep 5
```

Comment:
The way I've gone about this is to using a combination of `timeout` and grepping: https://github.com/rapidpro/rapidpro-docker/blob/master/.travis.yml#L45, not sure what you'd prefer. If `sleep 5` is enough to not cause random build failures then let's keep it as its simpler.